docs/deploying-to-azure-containerapps.md

```diff
@@ -730,6 +731,72 @@ az staticwebapp secrets list  \
 
+# Load Testing
+
+Now time to add some load to the application.
+This way, we will be able to see the auto-scaling in Azure Container Apps.
+
+To add some load to an application, you can do it locally using [JMeter](https://jmeter.apache.org), but you can also do it remotely on Azure using [Azure Load Testing](https://azure.microsoft.com/services/load-testing) and JMeter.
+Azure Load Testing is a fully managed load-testing service built for Azure that makes it easy to generate high-scale load and identify app performance bottlenecks.
+It is available on the [Azure Marketplace](https://azuremarketplace.microsoft.com).
+For that, we need to go the 
+
+To use Azure Load Testing, go to the [Azure Portal](https://portal.azure.com), search for the Marketplace and look for "_Azure Load Testing_"  in the Marketplace.
+Click on "_Create_":
+
+![load-testing-1-marketplace](../images/load-testing-1-marketplace.png)
+
+Create a load testing resource by giving it a name (`super-heroes-load-testing`), a location, and a resource group.
+Click on "Create":
+
+![load-testing-2-create](../images/load-testing-2-create.png)
+
+Creating a load testing resource can take a few moment.
+Once created, you should see the Azure Load Testing available in your resource group: 
+
+![load-testing-3-list](../images/load-testing-3-list.png)
+
+Select `super-heroes-load-testing` and click on "_Tests_" and then "Create".
+You can either create a quick load test using a wizard, or create a load test using a JMeter script.
+Choose this second option:
+
+![load-testing-4-upload-jmeter](../images/load-testing-4-upload-jmeter.png)
+
+Before uploading a JMeter script, create a load test by entering a name (eg. "_Make them fight_"), a description and click next: 
+
+![load-testing-5-create-test-1](../images/load-testing-5-create-test-1.png)
+
+Now that you are on the "Test plan" menu, you can upload the JMeter file (located under `scripts/jmeter/src/test/jmeter/fight.jmx`) as well as the `user.properties` file. 
+The JMeter file sets up a load campaign targetting the "Fight" microservice.
+Basically, it will invoke the `FightResource` endpoint so super heroes and super villains will fight.
+Before uploading the `user.properties` file, make sure you change the properties so you target the `FightResource` endpoint URL:
+
+```properties
+# Change these numbers depending on the load you want to add to the application
+LOOPS=20
+THREADS=2
+RAMP=1
+
+# Azure
+FIGHT_PROTOCOL=https
+FIGHT_PORT=443
+# Change the host depending on your settings
+FIGHT_HOST=rest-fights.kindocean-1cba89db.eastus.azurecontainerapps.io
+```
+
+![load-testing-5-create-test-2](../images/load-testing-5-create-test-2.png)
+
+Execute the test and you will get some metrics:
+
+![load-testing-6-stats](../images/load-testing-6-stats.png)
+
+Go back to the "_Home_" of your Azure portal and select the `rest-fights` app.
+Click on "_Metrics_".
+You will notice that the number of replicas has increased from 1 replica to 10.
```

Comment:
When I click on `metrics` I see something different than you do. Is there some configuration that needs to be done on the metrics screen?

![image](https://user-images.githubusercontent.com/363447/191089143-39a65eee-ff96-47fe-ad35-b6a932308830.png)